locales-pending/premium.ftl

```diff
@@ -23,6 +23,16 @@ toolbar-app-picker-product-fx-desktop = { -brand-fx-desktop }
 toolbar-app-picker-product-fx-mobile = { -brand-fx-mobile }
 toolbar-app-picker-by-mozilla = Made by { -brand-mozilla }
 
+user-menu-trigger-label = Open menu
+user-menu-trigger-tooltip = Profile
+user-menu-manage-fxa-label = Manage your { -brand-fxa }
```

Comment:
```suggestion
user-menu-manage-fxa-label = Manage your { -brand-fx-account }
```

We should use `-brand-fx-account`, which is correctly capitalized (and in the v2 section of the file)